presentation.ipynb

```diff
@@ -122,7 +122,8 @@
     " - `ndarray`: A generic, n-dimensional array data structure\n",
     " - Sophisticated machinery for operating on array data (vectorization, broadcasting, `ufuncs`)\n",
     " - Language extension/integration (C-API, `f2py`)\n",
-    "   * [Array API](https://docs.scipy.org/doc/numpy/reference/c-api.array.html) for accessing/extending array functionality in external libraries"
+    "   * [Array API](https://numpy.org/doc/1.17/reference/c-api.array.html) for accessing/extending array functionality in external libraries\n",
```

Comment:
I think numpy.org is now the canonical URL

Reply:
Thanks, nice catch

Reply:
Thanks, nice catch